CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# dbt_quickbooks v0.VERSION.VERSION
```

Comment:
if we are updating the packages.yml we gotta cut a release for the transform

Reply:
True, updated this